modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/components/form-renderer-custom-fields/RepeatableField.es.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React, {useContext} from 'react';
+
+export default ({AppContext, children, field, index}) => {
+	const [{focusedField}] = useContext(AppContext);
+
+	/**
+	 * Repeatable Behavior is not implemented to App Builder FieldSet yet !
+	 */
+
+	if (focusedField.type === 'fieldset') {
+		return <></>;
```

Comment:
@kevenleone could you use return null instead of the fragment?

Reply:
Sure, @interaminense